spec/enumerable_spec.rb

```diff
@@ -0,0 +1,80 @@
+require "./enumerable.rb"
+
+RSpec.describe Enumerable do 
+  let(:array) { [4,5,6] }
+  let(:my_each_arr) { array.my_each { |n| n } } 
```

Comment:
From this line up to line 8 are not necesary and should be removed, because we need the tests to be done in the test blocks and not pass the values we got from a variable declaration.